server/https.go

```diff
@@ -54,14 +54,16 @@ func New(bus *usb.USB) (*server, error) {
 	}
 	r := mux.NewRouter()
 
-	r.HandleFunc("/", s.Info)
-	r.HandleFunc("/configure", s.Info)
-	r.HandleFunc("/listen", s.Listen)
-	r.HandleFunc("/enumerate", s.Enumerate)
-	r.HandleFunc("/acquire/{path}", s.Acquire)
-	r.HandleFunc("/acquire/{path}/{session}", s.Acquire)
-	r.HandleFunc("/release/{session}", s.Release)
-	r.HandleFunc("/call/{session}", s.Call)
+	sr := r.Methods("POST").Subrouter()
+
+	sr.HandleFunc("/", s.Info)
```

Comment:
trezor-link requests / with GET

Reply:
yes, but trezor-link needs to be rewritten anyway for HTTP (I am doing that right now)

Reply:
I am rewriting trezor-link so that the two bridge versions are in a different source code (they are sufficiently different and it will make it easier to remove v1 later :))